wicket-core/src/test/java/org/apache/wicket/markup/html/form/upload/MultiFileUploadFieldTest.java

```diff
@@ -63,13 +64,16 @@ protected void onSubmit()
 				for (int i = 1; i < 2; i++)
 				{
 					FileUpload fileUpload = uploads.get(i);
-					assertEquals(MultiFileUploadFieldTest.class.getSimpleName()+i+".txt", fileUpload.getClientFileName());
+					assertEquals(MultiFileUploadFieldTest.class.getSimpleName() + (i - 1) + ".txt",
```

Comment:
I run the test with i-1 and it was success, but maybe the code at this place changed a bit. So if it passing revert it.

Thanks a lot!